NEXT-RELEASE.md

```diff
@@ -26,6 +26,19 @@ or compiled, even if it is not executed:
 
 -   The `-source` command is now deprecated. Use `eval` instead.
 
+-   The `eval-symlinks` command is now deprecated. Use `path:eval-symlinks`
```

Comment:
Hmm, I've seen both in release notes of other software and to me "is now deprecated" signifies that "this feature is deprecated since this release" (as opposed to already deprecated before).

Googling ["release notes" "is deprecated"](https://www.google.co.uk/search?q=%22release+notes%22+%22is+deprecated%22) and ["release notes" "is now deprecated"](https://www.google.co.uk/search?q=%22release+notes%22+%22is+now+deprecated%22) suggests that release notes do seem to say "is now deprecated" (178k vs 328k results).

Reply:
> I've seen both in release notes of other software...

Yes, because English is a slippery language with lots of quirks; perhaps more so than any other human language. However, in this case the "now" in "is now deprecated" formulation is redundant. If it was deprecated in an earlier release but not documented then this should say something like "was deprecated, albeit not formally documented, in version $whatever." But is obviously silly since it doesn't matter that the intent was to deprecate it in a prior release if it wasn't formally documented as being deprecated in that prior release. You can't tell users today to pretend you documented this was deprecated six months ago.

My name is Don Quixote :smile: